tests/test_tasks.py

```
@@ -0,0 +1,466 @@
+def _index_page(typst: str, page_id: str = "tasks") -> str:
+    needle = f"[{'Tasks'} <{page_id}>]"
+    for page in _pages(typst):
+        if needle in page and "rotate(" not in page:
+            return page
+    raise AssertionError(f"no Tasks index page {page_id}")
+
+
+def _week_page(typst: str, week_id: str) -> str:
+    marker = f"<tasks-{week_id}>"
+    for page in _pages(typst):
+        if marker in page and "rotate(" not in page and f"[Tasks <tasks-" not in page:
```

Ruff (0.16.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)